app/models.py

```diff
@@ -99,6 +99,14 @@ class Language(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String, unique=True, nullable=False)
 
+    @property
```

Comment:
we have a `serialize_language` method on `Resources` how do you see that tieing into this?

Reply:
Well, the thing about it is that it's a Many To Many relationship. So, in some contexts, we want a list of languages (as strings), and in other contexts we want the IDs included in the data. 

So, let's suppose you get all the languages (including IDs). Then you can take an ID and send a request to get all the resources associated with that. Or all the categories, etc..

Now, let's suppose you get a certain resource. You just want the languages associated with it, you don't necessarily need the IDs of those languages, do you?

So they serve different purposes.